src/test/java/io/meeds/qa/ui/stepDefinitions/AddUserStepDefinition.java

```diff
@@ -215,6 +215,16 @@ public void searchForSecondCreatedUser() {
     addUserSteps.searchForUsersByName(secondUserFirstName + ' ' + secondUserLastName);
   }
 
+  @And("^I search for first random created user$")
+  public void searchForFirstRandomUser() {
+    addUserSteps.searchForUsersByName(firstUserName);
+  }
+
+  @And("^I search for second random created user$")
+  public void searchForSecondRandomUser() {
+    addUserSteps.searchForUsersByName(secondUserName);
+  }
+
```

Comment:
```suggestion
  @And("^I search for (.*) random created user$")
  public void searchForRandomUser(String userPrefix) {
    String userName = Serenity.sessionVariableCalled(userPrefix + "UserName");
    addUserSteps.searchForUsersByName(userName);
  }
```

For more flexibility with all users.